avtozip/store/views.py

```diff
@@ -1,8 +1,26 @@
 """Store application views."""
 
+from django.http import HttpResponseRedirect
 from django.shortcuts import render
 
+from .api.resources import ProductResource
+from .forms import ProductFormSet
+
 
 def index_view(request):
     """INDEX view of store application."""
-    return render(request, 'store/index.html')
+    if request.method == 'POST':
+        formset = ProductFormSet(request.POST)
+        if formset.is_valid():
+            formset.save()
+            return HttpResponseRedirect(request.META.get('HTTP_REFERER'))
```

Comment:
We should not redirect here, just do nothing as it will be represented by `render`